data.txt missing

Hi,
I've installed FAASGS, the corresponding dependencies and a web server on my Raspberry Pi 3. I had some trouble with sats.sh and followed the steps in the other issue. I created the file /root/.predict/predict.qth and executed `/usr/bin/predict -t /etc/sat_data/data.txt -p NOAA19`. Then it shows the following error message:
`*** ERROR!  Your TLE file "/etc/sat_data/data.txt" could not be loaded!`
If I create a file called data.txt in /etc/sat_data, and run sats.sh once again, it shows some satellites but not the passes:
`Updating data and scheduling new sats
Can't fetch fresh TLE data, using old one
=> ISS (ZARYA)/ISS
=> RADFXSAT (FOX-1B)/AO91-FOX-1B
=> FOX-1CLIFF (AO-95)/AO95-FOX-1CLIFF
=> FOX-1D (AO-92)/AO92-FOX-1D
=> DIWATA-2B/PO101-DIWATA-2B
=> NOAA 19/NOAA19
=> NOAA 18/NOAA18
=> NOAA 15/NOAA15`

Thank you for your help in advance!

After running `make install` it shows the following error message:
`rm: removal of '/var/www/html/index.html' is not possible: File or directory not found
make: *** [Makefile:18: wwwt] Error 1`
The message was originally in German.

--- a/README.md
+++ b/README.md
@@ -67,6 +67,19 @@ sudo apt install predict
 
 For Armbian you need to compile it from source, you can get it from the [Predict home page](https://www.qsl.net/kd2bd/predict.html).
 
+After installing predict you need to do this aditional step to make it run properly (install some files):
+
+```sh
+sudo -i
+mkdir /root/.predict
+cd /root/.predict
+wget https://raw.githubusercontent.com/kd2bd/predict/master/default/predict.db
+wget https://raw.githubusercontent.com/kd2bd/predict/master/default/predict.tle
+wget https://raw.githubusercontent.com/kd2bd/predict/master/default/predict.qth
+# Next step only if using a Rasberry Pi board
+ln -s /root/.predict /home/pi/.predict
+```
+
 - **WXtoImage**
 
 This wonderful piece of software was deprecated by the original authors but a group of enthusiast keep it alive in the [Restored WXtoImage](https://wxtoimgrestored.xyz/) site.

